src/Xrefcheck/Scan.hs

```diff
@@ -118,7 +119,8 @@ data ScanErrorDescription
   | FileErr
   | ParagraphErr Text
   | UnrecognisedErr Text
-  deriving stock (Show, Eq)
+  deriving stock (Show, Eq, Generic)
+  deriving anyclass (NFData)
```

Comment:
You can derive `NFData` with standalone deriving if you prefer to follow the same style that in `Xrefcheck.Core` module, which contains a block like the following:

```haskell
instance NFData Position
instance NFData Reference
instance NFData AnchorType
instance NFData Anchor
instance NFData FileInfo
instance NFData RepoInfo

```